spec/helpers/posts_helper_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
```

Comment:
I meant removing the whole file and the associated helper file since they don't look used. The rails-helper part.